### PR TITLE
MS Tall Wall Update #002

### PR DESCRIPTION
**Pull request by scarwolff**
## About The Pull Request

Adds tall walls for mappers (not functional yet, we gotta wait on TG), also fixes the stupid smoothing system and adds a single tall door.

### CODE REVIEW

**Comment by uomo91** [comment]
Will try to playtest this soon, until then needs approval from Infra.

**Review by uomo91 — CHANGES_REQUESTED** [review]
Low walls are not getting proper `icon_state`'s.

**Comment by infraredbaron** [comment]
Brick wall and low brick wall doesn't match. Use proper walls, instead of crayon colored sandstone walls

**Review by infraredbaron — APPROVED** [review]
Sprites are good enough to serve their purpose - allow mappers to actually work with something.

File: mojave/structures/doors.dm
Comment (uomo91) [inline mojave/structures/doors.dm:43]: Make this a switch, and also I think this should be getting done in initialize(), I don't think the code needs to be rerun every single time the door is opened or closed, just when the door is initialized.

**Review by uomo91 — CHANGES_REQUESTED** [review]

**Review by uomo91 — CHANGES_REQUESTED** [review]
The doors are not animating properly.

Also these are missing sprites:
/turf/closed/wall/ms13/adobe
/obj/structure/table/low_wall/reinforced/bunker
Unclear to me if they're placeholders that don't have sprites yet or what.

**Review by uomo91 — CHANGES_REQUESTED** [review]
Doors are layering very oddly.

**Review by uomo91 — CHANGES_REQUESTED** [review]
The doors still just don't function, and it seems Infra has to make a small tweak to the low wall templates.

**Comment by Koshenko** [comment]
Door related code has been commented out until more work is done upstream on tall walls.  Leaving the door assets is fine for now.  Low walls reconfigured to properly smooth at 32x32.  Since this is mostly for the benefit of the mappers as a source of placeholders, I'm going to go ahead and merge this.

**Pull request merged**
